### ISSUE TITLE
No APK in 1.4.1?

### ISSUE THREAD

**Issue by licaon-kter** [MateusRodCosta/Share2Storage#44]
https://gitlab.com/fdroid/checkupdates-bot-fdroiddata/-/jobs/8500624082

**Comment by MateusRodCosta** [comment]
Hey, my bad, I have tagged a new release yesterday but ended up not going with the rest of the release process yesterday.

I was going over with publishing to Play Store and building an APK today.

**Comment by MateusRodCosta** [comment]
Sorry for the trouble, should be available now as https://github.com/MateusRodCosta/Share2Storage/releases/tag/v1.4.1

**Comment by licaon-kter** [comment]
https://gitlab.com/fdroid/checkupdates-bot-fdroiddata/-/jobs/8504946664#L676

We find that there is a `DependencyInfoBlock` in your APK.

It's a [Signing block](https://bi-zone.medium.com/easter-egg-in-apk-files-what-is-frosting-f356aa9f4d1) added by AGP and encrypted with the Google public key so it can't be read by anyone else except Google. You can read more about it [here](https://gitlab.com/fdroid/admin/-/issues/367) and [here](https://gitlab.com/fdroid/fdroidserver/-/issues/1056).

While this was added a while ago, we were only enforcing it for new apps, and recently we started scanning updates too.

Could you please disable it with the following code?

```
android {
    dependenciesInfo {
        // Disables dependency metadata when building APKs.
        includeInApk = false
        // Disables dependency metadata when building Android App Bundles.
        includeInBundle = false
    }
}
```

We will need a new release. Thanks!

**Comment by MateusRodCosta** [comment]
> We find that there is a DependencyInfoBlock in your APK.

I would guess this is part of https://support.google.com/googleplay/android-developer/answer/10358880?hl=en ?

> Could you please disable it with the following code?

I wonder if there's a benefit from still shipping that for Google Play, maybe I should starting building a fdroid flavor and a gplay flavor?

Or, maybe even simpler, only disable it for APKs but keep for bundles...

**Comment by licaon-kter** [comment]
> I wonder if there's a benefit from still shipping that for Google Play, maybe I should starting building a fdroid flavor and a gplay flavor?

that's up to you, eg. keep it for Play or in bundles

**Comment by licaon-kter** [comment]
> I would guess this is part of https://support.google.com/googleplay/android-developer/answer/10358880?hl=en ?

I don't see how it relates to this

**Comment by MateusRodCosta** [comment]
Released [1.4.1a](https://github.com/MateusRodCosta/Share2Storage/releases/tag/v1.4.1a)
